src/bindings/python/src/openvino/frontend/pytorch/torchdynamo/backend.py

```diff
@@ -49,18 +49,26 @@
 
 openvino_options = {}
 
-@register_backend
+
 @fake_tensor_unsupported
 def openvino(subgraph, example_inputs, options=None):
-    if (_get_aot_autograd(options)):
+    if _get_aot_autograd(options):
         global openvino_options
         openvino_options = options
         decompositions = _get_decompositions(options) + get_inf_decomposition_list() + get_aot_decomposition_list()
-        return aot_autograd(fw_compiler=fx_openvino,
-                            bw_compiler=fx_openvino,
-                            decompositions=get_decompositions(decompositions))(subgraph, example_inputs)
+        return aot_autograd(fw_compiler=fx_openvino, bw_compiler=fx_openvino, decompositions=get_decompositions(decompositions))(subgraph, example_inputs)
     return fx_openvino(subgraph, example_inputs, options)
 
+
+try:
+    from packaging import version
+
+    if version.parse(torch.__version__) < version.parse("2.5.0"):
+        register_backend(compiler_fn=openvino, name="openvino")
+except ImportError:
```

Comment:
may be it is better to check if `backend` is registered by this code?

```
if not hasattr(torch.backends, 'openvino') or not torch.backends.openvino.is_available()
```

that is torch version agnostic approach

Reply:
Agree, this can be improve. However I am not sure if the registered backends list can be easily accessible. Lets merge this PR as is and consider this improvement for future @suryasidd